src/ctap/hid/mod.rs

```diff
@@ -417,7 +417,7 @@ impl CtapHid {
     #[cfg(feature = "with_ctap1")]
     fn ctap1_success_message(cid: ChannelID, payload: &[u8]) -> HidPacketIterator {
         let mut response = payload.to_vec();
-        let code: u16 = ctap1::Ctap1StatusCode::SW_NO_ERROR.into();
+        let code: u16 = ctap1::Ctap1StatusCode::SW_INTERNAL_EXCEPTION.into();
```

Comment:
Here we need to either change the name or the error code.
A function called `ctap1_success_message` shouldn't expect an `INTERNAL_EXCEPTION` error code.

Reply:
the only excuse i can come up with is that i wasn't thinking straight. in my defense the line number tripped me off